pkgs/by-name/pr/proj-data/tests.nix

```diff
@@ -0,0 +1,21 @@
+{ runCommand, proj, proj-data }:
+
+let
+  inherit (proj-data) pname;
+in
+runCommand "${pname}-tests" { meta.timeout = 60; }
+  ''
```

Comment:
This test is using pipelines, so `set -o pipefail` might help.

Reply:
Good point, done.